src/python/antares_xpansion/problem_generator_driver.py

```diff
@@ -80,9 +78,7 @@ def set_output_path(self, output_path):
         if output_path.exists():
             self._output_path = output_path
             self.xpansion_output_dir = output_path.parent / \
-                (output_path.stem+"-Xpansion")
-            if self.xpansion_output_dir.exists():
-                shutil.rmtree(self.xpansion_output_dir)
+                                       (output_path.stem + "-Xpansion")
```

Comment:
This Pr is about creating -xpansion and lp dir so why you modify the python code?
else we face in some case memory issues and i think it's a good practice to clean the xpansion dir before the problem_generation run.